tables/uga.utb

```diff
@@ -0,0 +1,73 @@
+# liblouis: Ugaritic Grade 1 table
+#
+# ------------
+#-name: Ugaritic grade 1
+#-index-name: Ugaritic uncontracted
+#-display-name: Ugaritic uncontracted braille as used in the study of Ugaritic.
+#
+#+language:uga
+#+type:literary
+#+contraction:no
+#+grade:1
+#+system:uga
+
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
```

Comment:
Other tables have a `#-license` field to go along with the full license statement (`#-license: LGPLv2.1`), as well as one or more `#-copyright` fields.

Reply:
@paulGeoghegan @yeatersink I'm still missing some copyright lines in your tables.